.travis.yml

```diff
@@ -83,6 +83,25 @@ matrix:
         - USE_CATKIN=OFF
       services: docker
 
+    - os: linux
+      env:
+        - BUILD_NAME=BIONIC_CATKIN_FULL_RELEASE
```

Comment:
Should this be `BIONIC_CATKIN_FULL_RELEASE_AIKIDOPY`?

Reply:
Good catch! :+1: